examples/xt.lua

```diff
@@ -0,0 +1,49 @@
+--
+-- SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+-- SPDX-License-Identifier: MIT OR GPL-2.0-only
+--
+
+-- Match DNS packets using netfilter match
+
+local xt = require("xtable")
+local linux = require("linux")
+local action = xt.action
+local family = xt.family
+
+local udp = 0x11
+local dns = 0x35
+
+local function nop() end
+
+local function match_packet(skb)
+	print("match_packet called\n")
+	-- ip header
+	local ipb = skb:getuint8(0)
+	local ihlen = ((ipb << 4 ) & 0xFF) >> 4
+	local iphdrlen = ihlen * 4
+	local proto = skb:getuint8(9)
+
+	if proto == udp then
+		local srcport = linux.ntoh16(skb:getuint16(iphdrlen))
+		local dstport = linux.ntoh16(skb:getuint16(iphdrlen + 2))
+		if srcport == dns or dstport == dns then
+			print("DNS packet\n")
+			return action.ACCEPT
+		end
+	end
+
+	return action.DROP
+end
+
+local match_ops = {
+	name= "luaxtabletest",
```

Comment:
```suggestion
	name = "luaxtabletest",
```

please, mind the code style.. `"%s/\([a-z]\)=/\1 =/g"`

Reply:
I wouldn't name this "luaxtabletest".. I think you should use a more descriptive name indicating what your extension really does..